contextual_lenses/pfam_family_classification/pfam_utils.py

```diff
@@ -154,7 +173,7 @@ def compute_embeddings(encoder, data_batches):
 
 
 def pfam_nearest_neighbors_classification(encoder, train_family_accessions, test_family_accessions, batch_size=512, 
-                                          n_neighbors=1, train_samples=None, test_samples=None, restricted=False):
```

Comment:
The variable name 'restricted' is vague. What does it mean?

Reply:
not sure why this is here, this 'restricted' variable has been removed